packages/sources/gsr/src/transport/crypto-lwba.ts

```diff
@@ -0,0 +1,89 @@
+import { WebSocketTransport } from '@chainlink/external-adapter-framework/transports'
+import { BaseEndpointTypes } from '../endpoint/crypto-lwba'
+import { getToken } from './authutils'
+import { makeLogger, ProviderResult } from '@chainlink/external-adapter-framework/util'
+
+const logger = makeLogger('GSR WS LWBA price')
```

Comment:
Is there a precedent for having spaces in the logger name? I've usually seen snake case as a single word...

Reply:
i know it's outdated, but the GSR price endpoint was the inspiration
`const logger = makeLogger('GSR WS price')`